tensorflow_datasets/audio/commonvoice.py

```diff
@@ -0,0 +1,71 @@
+"""
+Mozilla Common Voice Dataset (English)
+"""
+import os
+import csv
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+_DOWNLOAD_LINK = {
+    "en": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/en.tar.gz"}
+_SPLITS = {
+    tfds.Split.TRAIN: "train",
+    tfds.Split.TEST: "test",
+    tfds.Split.VALIDATION: "validated"}
+
+
+class CommonVoice(tfds.core.GeneratorBasedBuilder):
+    """
+    Mozilla Common Voice Dataset (English)
+    """
+    VERSION = tfds.core.Version("1.0.0")
+
+    def _info(self):
+        return tfds.core.DatasetInfo(
+            description=("Mozilla Common Voice Dataset (English)"),
+            builder=self,
+            features=tfds.features.FeaturesDict({
+                "client_id": tfds.features.Text(),
+                "upvotes": tf.int32,
+                "downvotes": tf.int32,
+                "age": tfds.features.Text(),
+                "gender": tfds.features.Text(),
+                "accent": tfds.features.Text(),
+                "sentence": tfds.features.Text(),
+                "voice": tfds.features.Tensor(shape=(None,), dtype=tf.float32)
```

Comment:
Would it be possible to use the [Audio feature](https://www.tensorflow.org/datasets/api_docs/python/tfds/features/Audio)?

Reply:
@rsepassi can you please elaborate?

Reply:
You're using `Tensor`, but the voice is `Audio`, so the `Audio` feature may be able to help in that it knows how to read audio files. Please read the documentation on it.

Reply:
@rsepassi the Tensor formed is a result of the Audio Feature. it is formed by applying `encode_example()` on the mp3 files as you informed me on the gitter channel. Are you asking me to return an tfds.features.Audio() object for each mp3 file?